### PR TITLE
LPS-143024 We should search object field by the label

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-143024

### CODE REVIEW

**Comment by gabrielwas** [comment]
@marcelabc can we discuss about it?

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:reevaluate:1327401_2653

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:reevaluate:1357401_847

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectFieldModelDocumentContributor.java [hunk 1]
```diff
@@ -39,6 +40,11 @@ public void contribute(Document document, ObjectField objectField) {
 		document.addText(Field.NAME, objectField.getName());
 		document.addKeyword(
 			"objectDefinitionId", objectField.getObjectDefinitionId());
+		document.addLocalizedText(
+			"label",
+			LocalizationUtil.populateLocalizationMap(
+				objectField.getLabelMap(), objectField.getDefaultLanguageId(),
+				0));
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectFieldModelDocumentContributor.java:47]: @marcelabc could you reorder all of these method calls, please?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/query/contributor/ObjectFieldKeywordQueryContributor.java [hunk 2]
```diff
@@ -47,6 +47,8 @@ public void contribute(
 			booleanQuery, searchContext, Field.ENTRY_CLASS_PK, false);
 		_queryHelper.addSearchTerm(
 			booleanQuery, searchContext, Field.NAME, false);
+		_queryHelper.addSearchLocalizedTerm(
+			booleanQuery, searchContext, "label", false);
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/query/contributor/ObjectFieldKeywordQueryContributor.java:51]: @marcelabc can you reorder this one too? The `addSearchLocalizedTerm` comes first then the others.

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
I changed the label because I fixed the minor issues.

**Pull request closed without merging**